config/schemas/config.yaml

```
@@ -126,6 +363,658 @@ properties:
+                  enabled:
+                    default: false
+                    type: boolean
+                  ips:
+                    $ref: '#/$defs/iplist'
+                  ports:
+                    $ref: '#/$defs/portlist'
+                type: object
+            type: object
+        type: object
+      opensearch:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+          plugins:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+              ports:
+                $ref: '#/$defs/portlist'
+                default:
+                  - 443
+            type: object
+        type: object
+      rcloneSync:
+        additionalProperties: false
+        properties:
+          destinationObjectStorageS3:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+              ports:
+                $ref: '#/$defs/portlist'
+                default:
+                  - 443
+            type: object
+          destinationObjectStorageSwift:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+              ports:
+                $ref: '#/$defs/portlist'
+                default:
+                  - 5000
+            type: object
+          enabled:
+            default: true
+            type: boolean
+          secondaryUrl:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+              ports:
+                $ref: '#/$defs/portlist'
+                default:
+                  - 443
+            type: object
+        type: object
+      rookCeph:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: false
+            type: boolean
+        type: object
+      s3Exporter:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      thanos:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      velero:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+    type: object
+  nodeLocalDns:
+    additionalProperties: false
+    properties:
+      customConfig:
+        description: https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/#coredns-configmap-options
+        examples:
+          - |-
+            example.com:53 {
+              errors
+              cache 30
+              reload
+              loop
+              forward . 127.0.0.1:9005
+            }
+        type: string
```

[Our config for `nodeLocalDns.customConfig`](https://github.com/elastisys/compliantkubernetes-apps/blob/v0.37.0/config/config/common-config.yaml#L1090) is not set to anything (null), so I get this:
```
wc-config.yaml: nodeLocalDns.customConfig: Invalid type. Expected: string, given: null
```
This should fix it:
```suggestion
        type: 
          - string
          - "null"
```
Or if there is a better way to ignore it, since it is a optional configuration.

---

I'm generally against having things be nullable, as that is a different type according to most tools. So I see it kinda more as a bug that we haven't set the config to `""` by default to keep it empty.